test-infra/image-builder/roles/kubevirt-images/defaults/main.yml

```diff
@@ -20,6 +20,13 @@ images:
     converted: false
     tag: "latest"
 
+  ubuntu-2404:
+    filename: noble-server-cloudimg-amd64.img
+    url: https://cloud-images.ubuntu.com/noble/current/noble-server-cloudimg-amd64.img
```

Comment:
Is Ubuntu 24.04 need to use `QCow2 UEFI/GPT Bootable disk image with linux-kvm KVM optimised kernel` version? (Previously used the KVM version.)
Ubuntu has not yet released this version.

Reply:
UPD: it won't be released.
https://discourse.ubuntu.com/t/will-ubuntu-24-04-cloud-image-release-kvm-version/44626/2

@yankay Do you have any idea to resolve this CI?

Reply:
https://github.com/kubernetes-sigs/kubespray/pull/8958#issuecomment-1153823896

There is no `vm-ubuntu-2404` in Quay.
https://quay.io/organization/kubespray

Reply:
HI @tico88612 

I have using the "https://github.com/kubernetes-sigs/kubespray/tree/master/test-infra/image-builder" to create a new image in https://quay.io/repository/kubespray/vm-ubuntu-2404

Reply:
Because the image is using https://github.com/kubernetes-sigs/kubespray/pull/11167 .

Please remove the modify of file `test-infra/image-builder/roles/kubevirt-images/defaults/main.yml`.

Reply:
Got it. I will revert it.